components/org.wso2.carbon.identity.branding.preference.resolver/src/main/java/org/wso2/carbon/identity/branding/preference/resolver/UIBrandingPreferenceResolverImpl.java

```diff
@@ -82,20 +94,25 @@ public class UIBrandingPreferenceResolverImpl implements UIBrandingPreferenceRes
 
     private static final Log LOG = LogFactory.getLog(UIBrandingPreferenceResolverImpl.class);
     private final ExecutorService executorService = ThreadLocalAwareExecutors.newFixedThreadPool(1);
+    private static final String ORGANIZATION_DETAILS = "organizationDetails";
+    private static final String DISPLAY_NAME = "displayName";
 
     private final BrandedOrgCache brandedOrgCache;
+    private final BrandedAppCache brandedAppCache;
     private final TextCustomizedOrgCache textCustomizedOrgCache;
 
     /**
      * UI branding preference resolver implementation constructor.
      *
-     * @param brandedOrgCache Cache instance.
+     * @param brandedOrgCache        Cache instance for branded org.
      * @param textCustomizedOrgCache Cache instance for custom text.
+     * @param brandedAppCache        Cache instance for branded app.
```

Comment:
order is different

Reply:
Addressed with https://github.com/wso2-extensions/identity-branding-preference-management/pull/40/commits/a9322948eef77b783bd87838edceb408a676e7f4